.github/workflows/ci-failure-fix.yml

```
@@ -11,6 +11,8 @@ permissions:
   actions: read
   issues: write
   id-token: write
+  workflows: write
```

actionlint (1.7.10)

unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)